.gitignore

```diff
@@ -1,2 +1,3 @@
 .bundle
 _site/
+*~
```

Comment:
😄 I have `(setq backup-inhibited t)` in my init.el and never worried about these backup files again, but I guess it doesn't hurt to have it in .gitignore